test/mapped.test.js

```diff
@@ -51,7 +51,7 @@ describe('mapped coverage', function () {
         assert.equal(mc.fnMap[index].name,'(unknown_' + index + ')');
 
         index = mc.addBranch('if', loc(1, 15, 1, 20), [loc(1, 15, 1, 20), loc(1, 21, 1, 40)], [1,0]);
-        assert.ok(index);
+        assert.ok(index === 0);
```

Comment:
nit: why not just use `assert.equals` here? The assertion message if it fails would be more helpful.